module/API/Controller.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file Controller.h
+ * @brief モーター制御に用いる関数をまとめたラッパークラス
+ * @author takahashitom
+ */
+#ifndef CONTROLLER_H
+#define CONTROLLER_H
+
+#include "ev3api.h"
+#include "Motor.h"
+
+class Controller {
+ public:
+  static const int MOTOR_PWM_MAX = 100;
+  static const int MOTOR_PWM_MIN = -100;
+
+  Controller();
+
```

Comment:
```suggestion

  /**
   * コンストラクタ
   */
  Controller();

```
他のところはあるので。行頭のスペース考慮できてないので、一応修正後はclang-format通しておいてください。

Reply:
レビューありがとうございます。
コメント文の追加しておきました。

スリープに関しては、<unistd.h>中のsleep()が使えなさそうだったので、Timerクラスを定義し、その中でev3apiが提供しているsleepを使用することで解決しました。